src/Orleans/Messaging/SqlMembershipTable.cs

```diff
@@ -21,489 +21,148 @@ MIT License
-            "m.UpdateZone,m.FaultZone,m.SuspectingSilos,m.SuspectingTimes,m.StartTime,m.IAmAliveTime,m.ETag,v.[Version],v.ETag as VersionETag " +
-            "FROM [dbo].[OrleansMembershipTable] m LEFT JOIN [dbo].[OrleansMembershipVersionTable] v ON m.DeploymentId = v.DeploymentId WHERE m.DeploymentId = @deploymentid AND Address = @address AND Port = @port AND Generation = @generation";
-
-        private const string READ_VERSION = @"SELECT Version,ETag FROM [OrleansMembershipVersionTable] WHERE DeploymentId = @deploymentid";
-
-        private const string INSERT_ROW =
-            "INSERT INTO [OrleansMembershipTable] " +
-            "(DeploymentId,Address,Port,Generation,HostName,Status,ProxyPort,[Primary],RoleName,InstanceName,UpdateZone,FaultZone,SuspectingSilos,SuspectingTimes,StartTime,IAmAliveTime,ETag) " +
-            "VALUES (@deploymentid,@address,@port,@generation,@hostname,@status,@proxyport,@primary,@rolename,@instancename,@updatezone,@faultzone,@suspectingsilos,@suspectingtimes,@starttime,@iamalivetime,@newetag)";
-
-        private const string UPDATE_ROW =
-            "UPDATE [OrleansMembershipTable] " +
-            "SET Address = @address,Port = @port, Generation = @generation, HostName = @hostname, Status = @status, ProxyPort = @proxyport, [Primary] = @primary, RoleName = @rolename, InstanceName = @instancename, " +
-            "UpdateZone = @updatezone, FaultZone = @faultzone, SuspectingSilos = @suspectingsilos, SuspectingTimes = @suspectingtimes, StartTime = @starttime, IAmAliveTime = @iamalivetime, ETag = @newetag " +
-            "WHERE DeploymentId = @deploymentid AND Address = @address AND Port = @port AND Generation = @generation AND ETag = @etag";
-
-        private const string INSERT_VERSION_ROW =
-            "INSERT INTO [OrleansMembershipVersionTable] " +
-            "(DeploymentId,TimeStamp,Version,ETag) " +
-            "VALUES (@deploymentid,@timestamp,@version,@newetag)";
-
-        private const string UPDATE_VERSION_ROW =
-            "UPDATE [OrleansMembershipVersionTable] " +
-            "SET TimeStamp = @timestamp,Version = @version,ETag = @newetag " +
-            "WHERE DeploymentId = @deploymentid AND ETag = @etag";
-
-        private const string UPDATE_ALIVE_TIME =
-            "UPDATE [OrleansMembershipTable] " +
-            "SET IAmAliveTime = @iamalivetime, ETag = @newetag " +
-            "WHERE DeploymentId = @deploymentid AND Address = @address AND Port = @port AND Generation = @generation";
-
-        private const string DELETE_TABLE_ENTRIES =
-            "DELETE FROM [OrleansMembershipTable] " +
-            "WHERE DeploymentId = @deploymentid";
-
-        private const string DELETE_VERSION_ROW =
-            "DELETE FROM [OrleansMembershipVersionTable] " +
-            "WHERE DeploymentId = @deploymentid";
     }
-}
+}
```

Comment:
On Windows when using git and the line-ending used are CRLF, please add an extra linebreak to avoid this:

![wrong-eof](https://cloud.githubusercontent.com/assets/2031765/8147734/a9765ae0-1283-11e5-8864-2690ea9b7990.PNG)